bin/amphtml-update.py

```diff
@@ -426,8 +432,8 @@ def ParseRules(repo_directory, out_dir):
 					if 'I-AMPHTML-SIZER' == val:
 						continue
 
-					# The img tag is currently exclusively to transformed AMP, except as descendant of amp-story-player.
-					if 'IMG' == val and 'amp-story-player-allowed-descendants' != _list.name:
```

Comment:
`amp-story-player-allowed-descendants` was allowed to keep native `img` but spec adds it for transformed AMP 🤔 

https://github.com/ampproject/amphtml/blob/4ce3cd79520dbeaf5ed5364cbff58d3d71dee40e/extensions/amp-story-player/validator-amp-story-player.protoascii#L32